setup.py

```diff
@@ -5,39 +5,60 @@
 import os
 
 # copied from kymatio's setup.py: https://github.com/kymatio/kymatio/blob/master/setup.py
-plenoptic_version_spec = importlib.util.spec_from_file_location('plenoptic_version',
-                                                                'plenoptic/version.py')
+plenoptic_version_spec = importlib.util.spec_from_file_location(
+    "plenoptic_version", "plenoptic/version.py"
+)
 plenoptic_version_module = importlib.util.module_from_spec(plenoptic_version_spec)
 plenoptic_version_spec.loader.exec_module(plenoptic_version_module)
 VERSION = plenoptic_version_module.version
 
 setup(
-    name='plenoptic',
-    version='0.1',
-    description='Visual Information Processing',
-    license='MIT',
-    url='https://github.com/pehf/plenoptic',
-    author='LabForComputationalVision',
-    author_email='pef246@nyu.edu',
+    name="plenoptic",
+    version="0.1",
+    description="Visual Information Processing",
+    license="MIT",
+    url="https://github.com/LabForComputationalVision/plenoptic",
+    author="LabForComputationalVision",
+    author_email="pef246@nyu.edu",
```

Comment:
We should probably change this from Pierre's email, right? If we say the author is the lab, then we should have some lab-affiliated email here that we're okay getting contacted at.

We could also list all of us and all our emails: https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py

Not sure which is better

Reply:
Yeah was thinking the same. I'd rather not have my own email on it and would prefer perhaps a general lcv email or something.

Reply:
Removed author email altogether.